metaflow/plugins/kfp/tests/flows/flow_triggering_flow.py

```diff
@@ -68,7 +53,7 @@ def test_trigger_and_wait(self):
 
             print("\nTesting timeout exception for wait_for_kfp_run_completion")
             try:
-                run = wait_for_kfp_run_completion(run_id=run_id, wait_timeout=10)
+                wait_for_kfp_run_completion(run_id=run_id, wait_timeout=1)
```

Comment:
Can we put this as 0.1 instead of 1? Given it's just expecting to catch the `TimeoutError` anyway does the duration matter much, if we shave off 0.9 secs in theory at least we'd save that test runtime unless I'm missing some other detail 😄

Reply:
Updated.